apps/console/src/features/users/utils/user-management-utils.ts

```diff
@@ -28,21 +25,39 @@ export class UserManagementUtils {
      * Private constructor to avoid object instantiation from outside
      * the class.
      *
-     * @hideconstructor
      */
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     private constructor() { }
 
+    /**
+     * Resolves the username of the given user.
+     *
+     * @param user - User object.
+     * @returns The username of the given user as a string.
+     */
+    public static resolveUsername = (username: string): string => {
+        if (username.split("/").length > 1) {
+            username = username.split("/")[1];
+        }
+
+        return username;
+    };
+
     /**
      * Check if a user is the authenticated user.
      *
-     * @param {ProfileInfoInterface} authenticatedUser - Current authenticated user.
-     * @param {ProfileInfoInterface} evaluatingUser - Profile of the evaluating user.
-     * @return {boolean} If the user is the authenticated user or not.
+     * @param authenticatedUser - Currently logged in user
+     * @param user - User object.
+     * @returns Whether the given user is the current user or not.
      */
-    public static isAuthenticatedUser(authenticatedUser: ProfileInfoInterface,
-                                      evaluatingUser: ProfileInfoInterface): boolean {
+    public static isAuthenticatedUser = (authenticatedUser: string, username: string): boolean => {
```

Comment:
Better to verify the authenticated user with the user ID

Reply:
Resolved with 8413b271b71daf184e1b29a2ed740b1b046cdfd9